openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIRequestBodyMapper.java

```diff
@@ -48,22 +49,31 @@
 
 import javax.ws.rs.core.MediaType;
 
+import static io.ballerina.openapi.converter.Constants.APPLICATION_PREFIX;
+import static io.ballerina.openapi.converter.Constants.JSON_POSTFIX;
+import static io.ballerina.openapi.converter.Constants.OCTECT_STREAM_POSTFIX;
+import static io.ballerina.openapi.converter.Constants.TEXT_POSTFIX;
+import static io.ballerina.openapi.converter.Constants.TEXT_PREFIX;
+import static io.ballerina.openapi.converter.Constants.XML_POSTFIX;
+
 /**
  * OpenAPIRequestBodyMapper provides functionality for converting ballerina payload to OAS request body model.
  *
  * @since 2.0.0
  */
 public class OpenAPIRequestBodyMapper {
-    Components components;
-    OperationAdaptor operationAdaptor;
-    SemanticModel semanticModel;
+    private final Components components;
+    private final OperationAdaptor operationAdaptor;
+    private final SemanticModel semanticModel;
+    private final Optional<String> customMediaType;
```

Comment:
Shall make the type `Optional<String>` -> `String` and move the optional API into the attribute's usages, as optional API is mainly intended for return values from methods.